docs/supported_instruments.rst

```diff
@@ -127,3 +139,9 @@ TIMED/SEE
 
 .. automodule:: pysat.instruments.timed_see
    :members: __doc__
+
```

Comment:
Is this supposed to be incorporated in this pull request?

Reply:
Just an extra line. TIMED_SEE isn't actually new, just close to the new empty line.

Reply:
or do you mean the TIEGCM? It wasn't supposed to be in this pull, but apparently I had the code from my  recent TIEGCM pull in my branch.

Reply:
Yeah, that was what I meant.

Reply:
If you want to keep it in here, then let's wait to merge this until after the TIEGCM pull request.

Reply:
Fixed text.